internal/constants/constants.go

```diff
@@ -1,4 +1,4 @@
 package constants
 
-const SNYK_DEFAULT_API_URL = "https://app.snyk.io/api"
+const SNYK_DEFAULT_API_URL = "https://api.snyk.io"
```

Comment:
authentication from the IDEs always failed when not using `/api`. Are we sure, this is the correct default api?

Reply:
interesting, there was some confusion around the api url, which is why we consulted with the API teams. but worth testing,